lib/sourcemaps.js

```diff
@@ -71,14 +72,16 @@ exports.concatenateSourceMaps = function(sourceFilename, mapsWithOffsets, source
 
   // convert from library internals format to canonical
   var normalized = JSON.parse(JSON.stringify(generated));
+  var sourcePath;
 
   if (includeSourcesContent) {
     normalized.sourcesContent = normalized.sources.map(function(source) {
```

Comment:
Depends on the compilation path. Generally these are absolute URLs from Traceur, but otherwise no assumptions are made. 

This function just overrides existing `sourceContents` with actual file contents, if the source has an absolute path. Not assuming relative paths are relative to the `sourceRoot`, because that's the output path of the related source file, which transpilers and plugins are not aware of.

File contents taking precedence to included source is just a workaround for the Tracuer bug around `sourcesContent`.